socorro/scripts/fetch_crash_data.py

```diff
@@ -127,10 +197,17 @@ def main(argv):
         print('%s is not a directory. Please fix. Exiting.' % outputdir)
         return 1
 
-    api_token = os.environ.get('SOCORRO_API_TOKEN')
-    print('Using api token: %s' % api_token)
+    crashes = []
+    if args.crashid:
+        api_token = os.environ.get('SOCORRO_API_TOKEN')
+        print('Using api token: %s' % api_token)
```

Comment:
Can we please truncate this. Just in case I, or someone equally dangerous, copy all the stdout as part of a pastebin in a bug or a irc conversation or something.